docs/central-install-digital-ocean.rst

```diff
@@ -565,6 +565,104 @@ DKIM is a security trust protocol which is used to help verify mail server ident
+
+Central is not compatible with every kind of identity provider. Central uses the OpenID Connect (OIDC) protocol. When SSO is enabled in Central, Central does not manage passwords, but it still identifies users using their email address. Central assumes that the identity provider verifies email addresses, requiring users to prove ownership of the email address they specify. If that is not the case, then do not enable SSO in Central.
+
+If SSO is enabled, some features in Central will stop working. OData feeds will not be accessible in tools like Power BI or Excel. The API will not be available, which means that pyODK and ruODK will not be able to fetch data from Central. In the future, we hope to provide a way to access the API when SSO is enabled.
+
+To enable SSO in Central, you will need to provide information from your identity provider, specifically the issuer URL, client ID, and client secret. If you use Google Workspace or Azure Active Directory, you can follow the instructions below to find this information.
+
+#. In ``.env``, set ``OIDC_ENABLED`` to ``true``. Set ``OIDC_ISSUER_URL`` to the issuer URL that you obtained from your identity provider, ``OIDC_CLIENT_ID`` to the client ID, and ``OIDC_CLIENT_SECRET`` to the client secret.
+
+   .. code-block:: console
+
+     $ cd central
+
+   .. code-block:: console
+
+     $ nano .env
+
+   .. code-block:: bash
+
+     OIDC_ENABLED=true
+     OIDC_ISSUER_URL=my-issuer-url
+     OIDC_CLIENT_ID=my-client-id
+     OIDC_CLIENT_SECRET=my-client-secret
+
+#. Build and restart all containers.
+
+   .. code-block:: console
+
+     $ docker compose build && docker compose stop && docker compose up -d
+
+Two Accounts: Central and the Identity Provider
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you enable SSO, users will use their account on the identity provider to log into Central. However, users will still have a Central account that is separate from the identity provider. A Central account is not automatically created for each account on the identity provider. Instead, a Central administrator will need to create a Central account for each user of the identity provider who should be allowed to log into Central.
+
+Central users will be able to change their display name shown in Central and to choose a different name from what is shown in the identity provider. However, because Central identifies users by their email address, most users will not be allowed to change their email address. Only a Central Administrator will be able to change the email address associated with a Central account. That will be needed if a user's email address changes within the identity provider. In that case, an Administrator will need to manually change the user's email address within Central to match their new address in the identity provider.
```

Comment:
Email change is one of two special cases mentioned in the brainstorming doc. I didn't add the other case ("Email addresses change hands") because I'm not completely sure how to describe it.